tests/testthat/recorded_tests/rmd-prerendered/tests/mytest-expected/001.json

```diff
@@ -1,36 +1,43 @@
 {
   "input": {
     "bw_adjust": 1,
-    "n_breaks": "20"
+    "n_breaks": "20",
+    "n_breaks-selectized": ""
   },
   "output": {
     "eruptions": {
-      "src": "[image data sha1: 87a4f61ddd2d880d2d67631985bac379820e7732]",
+      "src": "[image data sha1: cebbfb5a3660a73cf0426844e057c40680785077]",
       "width": 738,
```

Comment:
Seems there's some randomness here that `ShinyDriver` can't capture? https://github.com/rstudio/shinytest/blob/b98f98c08a1b73a45768ab958c54cac6e1be08b5/tests/testthat/recorded_tests/rmd-prerendered/tests/mytest.R#L1